pkg/daemon/daemon.go

```diff
@@ -484,35 +485,34 @@ func ReexecuteForTargetRoot(target string) error {
 	targetBinBase := "run/bin/machine-config-daemon"
 	targetBin := filepath.Join(target, targetBinBase)
 
-	// Be idempotent
-	if _, err := os.Stat(targetBin); err != nil {
-		sourceBinary := "/usr/bin/machine-config-daemon" + sourceBinarySuffix
-		src, err := os.Open(sourceBinary)
-		if err != nil {
-			return fmt.Errorf("opening %s: %w", sourceBinary, err)
-		}
-		defer src.Close()
+	targetBinDir := filepath.Dir(targetBin)
+	if err := os.MkdirAll(targetBinDir, 0o755); err != nil {
```

Comment:
Since there was no `directoryExists` on 4.13 and looking at this I just changed to use `MkdirAll` like we do in other places

Reply:
I added directoryExist() in recent related PR  https://github.com/openshift/machine-config-operator/pull/3825/commits/edc68bedee8fadadc81137333f01bf6fd610f043. If we want we can cherry-pick that and 4.14 and 4.13 code will be consistent to the change